.azuredevops/pipeline-integration-test.yml

```diff
@@ -0,0 +1,196 @@
+
+            displayName: "Install Oracle Instant Client"
+
+          - task: ArchiveFiles@2
+            displayName: "Zip $(archiveName)"
+            inputs:
+              rootFolderOrFile: $(Build.SourcesDirectory)
+              includeRootFolder: false
+              archiveType: zip
+              archiveFile: $(Build.ArtifactStagingDirectory)/$(archiveName)
+              replaceExistingArchive: true
+
+          - publish: $(Build.ArtifactStagingDirectory)/$(archiveName)
+            displayName: "Publish Artifact $(archiveName)"
+            artifact: $(archiveName)
+
+  - stage: run_test
+    displayName: "Run integration test (${{ parameters.env }})"
+    dependsOn:
+      - prepare_env
+    pool: $(deploy-pool)
+    jobs:
+      - job:
+        displayName: "Run integration test (${{ parameters.env }})"
+        timeoutInMinutes: 0 # how long to run the job before automatically cancelling. When 0 is specified, the maximum limit is used
+        steps:
+          - checkout: none
+
+          - task: DownloadPipelineArtifact@2
+            displayName: "Download Artifact $(archiveName)"
+            inputs:
+              artifact: $(archiveName)
+              source: current
+
+          - task: ExtractFiles@1
+            displayName: "Unzip $(archiveName)"
+            inputs:
+              archiveFilePatterns: $(Pipeline.Workspace)/$(archiveName)
+              destinationFolder: $(Pipeline.Workspace)/$(destinationFolderExtractArtifact)
+              cleanDestinationFolder: true
+              overwriteExistingFiles: false
+
+          - script: |
+              sudo sh -c "echo $(Pipeline.Workspace)/$(destinationFolderExtractArtifact)/oracle/instantclient_21_6 > /etc/ld.so.conf.d/oracle-instantclient.conf"
+              sudo ldconfig
+
+            displayName: "Linux ldconfig Command for Oracle Instant Client"
+
+          - script: |
+              export PATH=/usr/bin:/usr/local/bin:$PATH #Add environment variables
+              echo 'Install Behave'
+              sudo pip3 install behave 
+              sudo pip3 install -U -r $(Pipeline.Workspace)/$(destinationFolderExtractArtifact)/requirements.txt #Install Requirements
+              echo python version
+              python3 -V
+              whereis behave
+              which behave
+              export PATH=/usr/local/bin/behave:/usr/bin:/usr/local/bin:$PATH #Add environment variables
+
+              #behave --junit-directory=$(Pipeline.Workspace)/$(destinationFolderExtractArtifact)/report --junit $(Pipeline.Workspace)/$(destinationFolderExtractArtifact)/src/integ-test/bdd-test/features/ --tags=runnable,midRunnable
+              behave --junit-directory=$(Pipeline.Workspace)/$(destinationFolderExtractArtifact)/report --junit $(Pipeline.Workspace)/$(destinationFolderExtractArtifact)/src/integ-test/bdd-test/features/Mod1/Priorità_Media/RPT_UNIVERSALE/T213_rptUniversale_carrello_noConvenzione_fasciaNessunaConv.feature --tags=midRunnable
```

Comment:
it appears that only a subset of tests are running , why ? 
should be something like `behave --junit-directory=$(Pipeline.Workspace)/$(destinationFolderExtractArtifact)/report --junit $(Pipeline.Workspace)/$(destinationFolderExtractArtifact)/src/integ-test/bdd-test/`

Reply:
We're still working on the branch, so PR is taking all the changes we make on the branch.

This will be the final command:
behave --junit-directory=$(Pipeline.Workspace)/$(destinationFolderExtractArtifact)/report --junit $(Pipeline.Workspace)/$(destinationFolderExtractArtifact)/src/integ-test/bdd-test/features/ --tags=runnable